packages/sado-connect/package.json

```diff
@@ -56,7 +56,8 @@
     "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
     "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
     "@headlessui/react": "^1.7.15",
-    "@sadoprotocol/ordit-sdk": "^0.0.4",
-    "boring-avatars": "^1.10.1"
+    "@sadoprotocol/ordit-sdk": "^0.0.7",
+    "boring-avatars": "^1.10.1",
+    "vite-plugin-css-injected-by-js": "^3.1.2"
```

Comment:
why did you add `"vite-plugin-css-injected-by-js"` as deps and not devdeps

Reply:
Oops. Fixed.